bears/vcs/git/GitCommitBear.py

```diff
@@ -260,6 +268,25 @@ def check_body(self, body,
                                'Commit body lines should not exceed {} '
                                'characters.'.format(body_line_length))
 
```

Comment:
E128 continuation line under-indented for visual indent'

*PycodestyleBear (E128), severity NORMAL, section `autopep8`.*